stl/inc/xutility

```diff
@@ -2005,6 +2004,20 @@ _NODISCARD constexpr size_t size(const _Ty (&)[_Size]) noexcept {
     return _Size;
 }
 
+#if __HAS_CXX20
+template <class _Container>
+[[nodiscard]] constexpr auto ssize(const _Container& _Cont)

```

Comment:
`_NODISCARD`, unlike `_INLINE_VAR`, has a purpose beyond simply language mode compat: `_NODISCARD` can be disabled by defining `_HAS_NODISCARD` to `0`. Since we spam it on everything under the sun, we need to have that escape hatch for the day when we make a mistake and release a `_NODISCARD` that triggers false positives.
```suggestion
_NODISCARD constexpr auto ssize(const _Container& _Cont)
```

Reply:
Done

Reply:
Done